go/vt/vtctld/api.go

```diff
@@ -251,7 +251,12 @@ func initAPI(ctx context.Context, ts topo.Server, actions *ActionRepository, rea
 		for _, keyspaceName := range keyspaceNamesList {
 			err := addSrvkeyspace(ctx, ts, cell, keyspaceName, srvKeyspaces)
```

Comment:
I think this function is useless and somebody was too eager to write it in the first place.

Can you please inline it here?

This way, your code here will be less indirect and more clear.

Change itself LGTM.